Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -525,27 +485,34 @@ function NotNullAssert(v: unknown): asserts v is NotNull {}
                         let index = -1;
                         if (input.value != '')
                         {
-                          index = select.findItem(input.value);
+                            index = select.findItem(input.value);
                         }
                         else
                         {
-                          const selectedItem = select.selected(true);
-                          const isAnnotationSelected = selectedItem != null && selectedItem.data.IsAnnotation === true;
-                          if (isAnnotationSelected)
+                            const selectedItem = select.selected(true);
+                            const isAnnotationSelected = selectedItem != null && selectedItem.data.IsAnnotation === true;
+                            if (isAnnotationSelected)
                             index = select.findItemPositionWhere (function (data) { return data === selectedItem });
```

Comment:
indent.

please check all the indent changes, there seems to be a lot of inconsistencies now.